kitsune/bundles.py

```diff
@@ -282,6 +291,34 @@
         ),
         'output_filename': 'community-min.js'
     },
+    'community-new-questions': {
+        'source_filenames': (
+            # This uses the minified version because it is optimized to leave
+            # out lots of debug stuff, so it is significantly smaller than
+            # just minifying react.js.
+            # TODO: Figure out how to include the full sized version in dev,
+            # because it produces much nicer error messages.
+            'bower/react/react.min.js',
+            # 'bower/react/react.js',
+            'bower/pikaday/pikaday.js',
```

Comment:
I think there used to be a `static/bower/`, before a bunch of rebases. That being said, I'm puzzled why this worked for me. *shrug*